src/features/lightspeed/errors.ts

```diff
@@ -184,11 +184,18 @@ ERRORS.addError(
     "Could not find an API Key for IBM watsonx Code Assistant. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  403,
+  new Error(
+    "error__no_default_model_id",
+    "Ansible Lightspeed does not have a model configured. Contact your Ansible administrator to configure a model, or provide a model id override in your Ansible extension settings.",
```

Comment:
Do we want to use the same wording here as the other message, `or specify a model in your Ansible extension settings under Lightspeed: Model Id Override.`?

Reply:
The message was updated in the latest commit.